docs/program/apis/sessions.md

```diff
@@ -57,37 +65,17 @@ To do so, add Raw JSON to the configuration of your robot in this format:
 
 The default heartbeat window is 2 seconds if this configuration is omitted.
 
-If you choose to use the session management API to manage sessions, your client must send at least one session heartbeat within the window you set.
-As soon as your window lapses or expires, the server will safely stop all resources that are marked for safety monitoring that have been last used by that session, and no others.
-A lapsed client will attempt to establish a new session immediately prior to the next operation it performs.
-
-## Usage
-
-Usage of the session management API differs across [Viam's SDKS](/program/).
-
-### Access the session management API
-
-To manage sessions on-robot manually, you can use the following client SDKs:
-
-{{< tabs >}}
-{{% tab name="Go" %}}
-
-To enable the [Session Management API](https://pkg.go.dev/go.viam.com/rdk/session) the Go Client SDK provides, [disable the default behavior of sessions](#disable-default-session-management).
+If you manually manage sessions, each client must send at least one heartbeat within the window you set.
 
-{{% /tab %}}
-{{% tab name="Other SDKs" %}}
+## Manage sessions with the session management API
 
-The session management API is not currently provided in the Python or TypeScript SDKs.
+The [Session Management API](https://pkg.go.dev/go.viam.com/rdk/session) is not currently provided in the Python or TypeScript SDKs.
 Use the Go Client SDK instead.
 
-{{% /tab %}}
-{{% /tabs %}}
-
-### Use the session management API to manually manage sessions
+To manage your sessions with the session management API:
```

Comment:
```suggestion
To manage your session with the session management API:
```
(opt) Maybe change to singular, re. past feedback just to keep all the same (it's hard to just say session or sessions though)